beerus_core/src/lightclient/ethereum/helios_lightclient.rs

```diff
@@ -57,6 +57,11 @@ impl EthereumLightClient for HeliosLightClient {
         self.helios_light_client.get_code(address, block).await
     }
 
+    async fn get_transaction_count(&self, address: &Address, block: BlockTag) -> Result<u64> {
+        // TODO: Rename after it has been renamed https://github.com/a16z/helios/pull/166#issuecomment-1379587761
+        self.helios_light_client.get_nonce(address, block).await
```

Comment:
Putting a todo here as this will be renamed on the next major version change [according](https://github.com/a16z/helios/pull/166#issuecomment-1379587761) to helios team

Reply:
Ok!